test/distributed/org/apache/cassandra/distributed/test/SnapshotsTest.java

```diff
@@ -469,4 +526,13 @@ private void exoticSnapshotNamesInternal(String[] exoticSnapshotNames)
             waitForSnapshot(tag, true, true);
         }
     }
+
+    private List<String> extractSnapshots(String listSnapshotsStdOut)
+    {
+        return Arrays.stream(listSnapshotsStdOut.split("\n"))
+                     .filter(line -> !line.isEmpty())
+                     .filter(line -> !line.startsWith("Snapshot Details:") && !line.startsWith("There are no snapshots"))
```

Comment:
Curious: Is there any benefit from adding two filters with a `&&` operator as opposed to four single filters on this stream?